lib/libc/glibc/sysdeps/generic/elf-initfini.h

```diff
@@ -0,0 +1 @@
+#define NO_INITFINI 1
```

Comment:
Upstream glibc sets this to zero in this file.

Reply:
Alright, will change

Reply:
Done

Reply:
Sorry, I should have been clearer earlier. As a general principle, I think it's better to copy the file 1:1 from upstream glibc when we don't need any changes to it (which seems to be the case here), so there are less diffs to review when upgrading glibc.